[2021-12-22 17:34:59,947] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:21:00+00:00 [queued]>
[2021-12-22 17:34:59,952] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:21:00+00:00 [queued]>
[2021-12-22 17:34:59,952] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:34:59,952] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 17:34:59,952] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:34:59,961] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): sketch> on 2021-12-20 00:21:00+00:00
[2021-12-22 17:34:59,964] {standard_task_runner.py:52} INFO - Started process 28253 to run task
[2021-12-22 17:34:59,968] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'sketch', 'scheduled__2021-12-20T00:21:00+00:00', '--job-id', '138', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmpio8_ryxk', '--error-file', '/tmp/tmpu1wdmgwa']
[2021-12-22 17:34:59,969] {standard_task_runner.py:77} INFO - Job 138: Subtask sketch
[2021-12-22 17:34:59,996] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:21:00+00:00 [running]> on host myamya
[2021-12-22 17:35:00,022] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=sketch
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:21:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:21:00+00:00
[2021-12-22 17:35:00,022] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 17:35:00,022] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sketch:0.4 /bin/bash /home/myamya/project/init.sh ']
[2021-12-22 17:35:00,026] {subprocess.py:85} INFO - Output:
[2021-12-22 17:35:20,598] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 17:35:20,635] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=sketch, execution_date=20211220T002100, start_date=20211222T083459, end_date=20211222T083520
[2021-12-22 17:35:20,660] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 17:35:20,678] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
